### ISSUE TITLE
Loss is decrease slowly

### ISSUE THREAD

**Issue by jinfagang** [zzh8829/yolov3-tf2#30]
![image](https://user-images.githubusercontent.com/21303438/60405602-ca92a380-9be3-11e9-86de-8d170e4f3104.png)

As you may notice, the loss seems unnormal after several epochs. it seems total loss does not converge.

**Comment by zzh8829** [comment]
can you share the command you executed for this

**Comment by jinfagang** [comment]
@zzh8829  I just using keras fit method to train

**Comment by Pari-singh** [comment]
Yeah, same. And I don't even know the reason why the loss values are so high and when do they exactly converge.

**Comment by zzh8829** [comment]
can you paste training data please I can't debug without data

**Comment by AnaRhisT94** [comment]
@zzh8829  I'm getting `nan`s in my case. Having 2 classes, cars and plates. it works perfect on Darknet and when I transfer to TensorFlow it just detects cars.

**Comment by zzh8829** [comment]
take a look at this https://github.com/zzh8829/yolov3-tf2/blob/master/docs/training_voc.md
